clang/test/CodeGen/cheri/riscv/non-cheri-march.c

```diff
@@ -0,0 +1,11 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py
+/// Using __capability without the appropriate -march string previously
+/// crashed clang during codegen instead of emitting an error
+// RUN: %clang_cc1 -triple riscv64 -o - -emit-llvm %s -verify=no-cheri
+// RUN: %clang_cc1 -triple riscv64 -target-feature +xcheri -o - -emit-llvm %s | FileCheck %s
+
+// CHECK-LABEL: @test(
+// CHECK-NEXT:  entry:
+// CHECK-NEXT:    ret i8 addrspace(200)* null
+//
+void *__capability test(void) { return 0; } // no-cheri-error{{__capability attribute is not supported on targets missing CHERI; specify an appropriate -march= or -mcpu=}}
```

Comment:
This pre-existing error message seem quite good but I can add a more specific one if that's preferred?